### DISCUSSION TITLE
Add version creation date to crates.io index

### DISCUSSION THREAD

**Discussion by viniciusd** [rust-lang/crates.io#4790] [Feature Requests]
I wonder if it would be possible to add the dates when versions were created to the [crates.io index](https://github.com/rust-lang/crates.io-index).

I am performing some time analysis on the data, but I am currently relying on the experimental daily dumps rust-lang/crates.io#3617. I am not sure how experimental they are still or if it is a solution that will stick, but it would be good to have the index as an alternative.

**Comment by Turbo87** [comment]
the main purpose of the index is to make it possible for the dependency resolver to work correctly. since the resolver does not need the creation date it is unlikely for us to add it.
